app/src/main/java/com/github/leuludyha/ibdb/presentation/components/reading_list/controls/ReadingStateControl.kt

```diff
@@ -13,23 +13,31 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import com.github.leuludyha.domain.model.library.Mocks
 import com.github.leuludyha.domain.model.library.Work
 import com.github.leuludyha.domain.model.user.UserPreferences
 import com.github.leuludyha.domain.model.user.WorkPreference
+import com.github.leuludyha.domain.util.TestTag
+import com.github.leuludyha.domain.util.testTag
 import com.github.leuludyha.ibdb.R
 import com.github.leuludyha.ibdb.ui.theme.IBDBTheme
 
+object TestTags {
```

Comment:
Cool object declaration, I like that. It is not what I have been doing though (just giving a tag to each node directly in the code). Maybe we could all agree on one way of declaring these tests tag so that they are more uniform. Nevertheless, I have seen in your previous pull requests that you formatted test tags as composable_name::test_tag_name, so I would propose to keep doing it that way (with _ instead of - and indicating the composable name before), I followed that convention (copied from you) during this sprint.